cs_scanimage.py

```diff
@@ -15,11 +15,13 @@
     'us-1': 'container-upload.us-1.crowdstrike.com',
     'us-2': 'container-upload.us-2.crowdstrike.com',
     'eu-1': 'container-upload.eu-1.crowdstrike.com',
+    'us-gov-1': 'container - upload.laggar.gcw.crowdstrike.com',
```

Comment:
The spacing in between `-` doesn't look right to me.

Reply:
Hm, let me punish nasty lisp in my emacs.